### PR TITLE
Initial build should set `requestPath` to `null`

### PR DESCRIPTION
**Pull request by leo**
This is required for https://github.com/zeit/now-builders/pull/410 and sets the `requestPath` property that is passed to Builders to `null` for initial builds.

### CODE REVIEW

File: src/commands/dev/lib/dev-builder.ts [hunk 1]
```diff
@@ -152,7 +152,7 @@ export async function executeBuild(
     })
   );
 
-  match.buildResults.set(requestPath, result);
+  match.buildResults.set(requestPath || '', result);
```
Comment (TooTallNate) [inline src/commands/dev/lib/dev-builder.ts]: I think we shouldn't default to `''` here. Just update the `BuildMatch` in types.ts to support `null` as a key for the map.

Comment (leo) [reply src/commands/dev/lib/dev-builder.ts]: Good call. Fixed!

File: src/commands/dev/lib/dev-server.ts [hunk 2]
```diff
@@ -142,15 +142,15 @@ export default class DevServer {
...
+        if (requestPath === null || await shouldServe(match, this.files, requestPath)) {
           this.triggerBuild(
             match,
             requestPath,
             null,
             result,
             filesChangedArray,
             filesRemovedArray
-          ).catch(err => {
+          ).catch((err: NodeJS.ErrnoException) => {
```
Comment (TooTallNate) [inline src/commands/dev/lib/dev-server.ts]: Why this change?

Comment (leo) [reply src/commands/dev/lib/dev-server.ts]: @TooTallNate It failed and said it had `any` type

**Review by TooTallNate — APPROVED** [review]
